### ISSUE TITLE
Required IAM roles are not documented

### ISSUE THREAD

**Issue by aaron-lane** [terraform-google-modules/terraform-google-sap#4]
The READMEs for the SAP HANA submodules list only `roles/storage.objectViewer` as required for the service account attached to the VM instances, but I have a client who has identified `roles/logging.admin` and `roles/compute.admin` as required roles to solve errors seen during execution of the startup script. It's possible that roles less privileged than "admin" may be sufficient, but this needs to be investigated.

**Comment by evkuzin** [comment]
As I understand from this script https://github.com/terraform-google-modules/terraform-google-sap/blob/d1b227de9416821f7986273597b5cb9d71894746/modules/sap_hana/sap_hana_python/main.tf#L2
All required roles contain this block:
```
'serviceAccounts': [{
                  'email': service_account,
                  'scopes': [
                      'https://www.googleapis.com/auth/compute',
                      'https://www.googleapis.com/auth/servicecontrol',
                      'https://www.googleapis.com/auth/service.management.readonly',
                      'https://www.googleapis.com/auth/logging.write',
                      'https://www.googleapis.com/auth/monitoring.write',
                      'https://www.googleapis.com/auth/trace.append',
                      'https://www.googleapis.com/auth/devstorage.read_write'
                      ]
}]
```
I will add PR soon

**Comment by morgante** [comment]
@evkuzin Those are scopes, not roles.

**Comment by evkuzin** [comment]
@morgante  You a right, my mistake. 
All what script do could be found here:
https://storage.googleapis.com/sapdeploy/dm-templates/sap_hana/startup.sh
And I found that script run this commands:
```
${GCLOUD} compute instances list --filter="name=('${host}')" --format "value(zone)"
${GCLOUD} --quiet logging write "${HOSTNAME}" "${HOSTNAME} Deployment \"${log_entry}\"" --severity=INFO  ${GCLOUD} --quiet compute instances add-metadata "${host}" --metadata "ssh-keys=root:$(cat ~/.ssh/id_rsa.pub)"  --zone "${host_zone}"
${GCLOUD} --quiet compute --project "${VM_NETWORK_PROJECT}" addresses create "${HOSTNAME}" --addresses "${VM_IP}" --region "${VM_REGION}" --subnet "${VM_SUBNET}"
${GCLOUD} --quiet compute instances remove-metadata "${HOSTNAME}" --keys "${key}"
```

So script:
- list instances
- send logging
- add metadata
- create address
- remove metadata

This all could be allowed by these roles:
- roles/stackdriver.resourceMetadata.writer
- roles/compute.networkAdmin
- roles/logging.logWriter
and roles/storage.objectViewer because we already have it in README

So what would be the best way - add these 3 roles or just recommend to create a custom role with only needed rights, because f.e. compute.networkAdmin is wider than needed.

**Comment by aaron-lane** [comment]
@evkuzin the base requirement is to document the list of predefined roles which provide the necessary permissions and contain the least amount of irrelevant permissions, but if you have identified the specific permissions necessary then those can also be documented for users who create custom roles.

**Comment by aaron-lane** [comment]
Fixed by #5.
